File: gcd.py

def gcd(a, b):
    while b != 0:
        a, b = b, a % b
    return a
#get input numbers from the math trying to crack gcd.
num1 = int(input("enter an integer value:"))
num2 = int(input("enter another integer value:"))
result = gcd(num1, num2)
#Calaculate the gcd and print your result.
print("The greatest common divisor of",num1 ,"and",num2 ,"is", result)

File: gcd.c
#include<stdio.h>
#include<math.h>
int gcd(int a, int b){
while(b != 0){
int temp = b;
b=a%b;
a = temp;
}
return a;
 }
int main(){
int num1,num2;
printf("Enter the two numbers:");
scanf("%d  %d",&num1,&num2);
int result = gcd(num1, num2);
printf("The GCD of %d and %d is %d\n",num1,num2,result);
getchar();
}
